core/mavlink_commands.h

```diff
@@ -53,30 +54,27 @@ class MavlinkCommands
     const MavlinkCommands &operator=(const MavlinkCommands &) = delete;
 
 private:
-    enum class State {
-        NONE,
-        WAITING,
-        IN_PROGRESS,
-        DONE,
-        FAILED
-    } _state = State::NONE;
-    std::mutex _state_mutex {};
+    static constexpr double DEFAULT_TIMEOUT_NORMAL_S = 0.5;
```

Comment:
We need to find a way to get the default timeouts into the documentation (if they aren't already there), this is important behavior that should be visible to external developers

Reply:
@julianoes @mrpollo Arguably some timeouts should actually be part of the settable public API - and if so they could then be documented in the same way as everything else (?).

@julianoes Are all timeouts named consistently so we can find them? (i.e. can I have a list)

Reply:
Ok, I agree. Let's check this later.